src/main/java/ui/MenuControl.java

```diff
@@ -353,6 +353,17 @@ private void updateRepoRemoveList(Menu remove) {
             removeItem.setOnAction(e1 -> onRepoRemove(repoId));
             remove.getItems().add(removeItem);
         }
+
+        // Supposedly, we would like the menu not to close when the disabled MenuItem-s
+        // below are clicked. But this is a JDK bug; we can use CustomMenuItem.setHideOnClick(false)
+        // if we want to. The bug is that it only works for ContextMenu and not Menu (which
+        // we are using).
+        for (String usedRepoId : currentlyUsedRepos) {
+            MenuItem disabledRemoveItem = new MenuItem(usedRepoId + " (in use; non-removable)");
```

Comment:
I don't think the semicolon there looks nice =P

Maybe something shorter like just `(In Use)`?